routes/web/hardware.php

```diff
@@ -146,9 +146,9 @@ function () {
             [AssetFilesController::class, 'show']
         )->name('show/assetfile');
 
-        Route::delete('{assetId}/showfile/{fileId}/delete',
```

Comment:
This will break file deletion on assets. We already have a delete route - the one we use for table-button deletes. No route changes should be required here, unless I'm missing something (and I might be)

Reply:
I can double check we still use this. With the rewrite it probably isn't used. but I can look at it this weekend.

Reply:
We don't work on weekends :) It will keep until monday :D

Reply:
that was old, and couldn't get it to work. but it should be gone off the final bit